testcases/ock_tests.sh.in

```diff
@@ -316,7 +316,7 @@ main_script()
             init_slot $i || { echo "SKIPPING slot $i"; exit; }
         fi
         for item in $OCK_TRACE_TOKENS; do
-            [[ "$i" == "$item" ]] && export OPENCRYPTOKI_TRACE_LEVEL=4
+            [[ "$i" == "$item" ]] && export OPENCRYPTOKI_TRACE_LEVEL="${OCK_TRACE_LEVEL:-4}"
```

Comment:
Is the minus sign (dash) intended?

Reply:
Yes, this is the syntax for getting a variable and assign a default value when the variable is not set :-)